adks/skywater-130nm/README.md

```diff
@@ -0,0 +1,12 @@
+README
+==========================================================================
+Author : Alex Carsello
+Date   : July 12, 2022
+
+This ASIC design kit uses Skywater 130nm.
+
+More information is available in the README of the standard base kit
+located here:
+
+- https://github.com/mflowgen/skywater-130nm
```

Comment:
Can we mention here what we discussed about how we will handle versioning for now? The repo is currently set up as a set of pre-built versions of the ADK. This node will pull the corresponding version of the ADK's standard view.

Maybe a "version" parameter which defaults to "v0.0.0" for now?

Reply:
yes that's what I'm doing

Reply:
Done. For some reason the folder in the downloaded zip file is called skywater-130nm-0.0.0 and not skywater-130nm-v0.0.0

Reply:
Great! Can we just put what I said into a comment at the top of this configure.yml? and then I think we are done here.